nwbinspector/nwbinspector.py

```diff
@@ -112,7 +112,7 @@ def inspect_all(
 
     in_path = Path(path)
     if in_path.is_dir():
-        nwbfiles = list(in_path.glob("*.nwb"))
+        nwbfiles = list(in_path.glob("**/*.nwb", recursive=True))
```

Comment:
```suggestion
        nwbfiles = list(in_path.glob("**/*.nwb"))
```

Doesn't take this potion - looks like it's recursive over sub-directories by default with the `**/`: https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob